federation/cluster/federation-up.sh

```diff
@@ -82,7 +82,8 @@ function init() {
       "${FEDERATION_NAME}" \
       --host-cluster-context="${HOST_CLUSTER_CONTEXT}" \
       --dns-zone-name="${DNS_ZONE_NAME}" \
-      --image="${kube_registry}/hyperkube-amd64:${kube_version}"
+      --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
+      --apiserver-arg-overrides="--storage-backend=etcd2"
```

Comment:
Then why is this PR proposing to default ``kubefed`` to default to etcd3?  That would seem to risk breaking users.